src/app/EmployeesHistory/Components/itemsList.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const ItemsList = props => {
+  const listItems = props.items.map((value, index) => {
+    return (
+      <li key={index}>
```

Comment:
It's good that you added a key on the <li>, but  be careful because using the items index may cause some issues then changing the arrays content.
I would recomand using a library for generating keys like uuid.